src/main/java/io/pravega/connectors/flink/PravegaConfig.java

```diff
@@ -137,9 +137,7 @@ public PravegaConfig withTrustStore(String trustStore) {
      * @param scope The scope to use (with lowest priority).
      */
     public PravegaConfig withDefaultScope(String scope) {
```

Comment:
I prefer change this into `withScope` and the variable into `scope`. But this would be breaking change, is there any concern on this? @maddisondavid

Reply:
To keep compatibility, we keep the original and add a new `withScope` to do this.